Header header

string rx_address
string tx_address
float64 rx_level
float64 link_budget
string channel_band
string channel_class
int64 counter
